akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -584,6 +587,7 @@ object HttpEntity {
   /**
    * INTERNAL API
    */
+  @InternalApi
```

Comment:
This is fixed, I don't know why it is still showing up. I guess Github UI got confused...